tools/sdk-cli/src/main.rs

```diff
@@ -5,32 +5,38 @@ mod commands;
 mod config;
 mod persist;
 
-use anyhow::{anyhow, Result};
+use crate::command_handlers::CliHelper;
+use anyhow::{anyhow, ensure, Result};
 use breez_sdk_core::BreezServices;
 use clap::Parser;
 use command_handlers::handle_command;
 use commands::{Commands, SdkCli};
 use persist::CliPersistence;
 use rustyline::error::ReadlineError;
+use rustyline::hint::HistoryHinter;
 use rustyline::Editor;
 use std::path::Path;
 
 #[tokio::main]
-async fn main() {
+async fn main() -> Result<()> {
     let cli = SdkCli::parse();
     let data_dir = cli.data_dir.clone().unwrap_or(".".to_string());
     let data_dir_path = Path::new(&data_dir);
-    if !data_dir_path.exists() {
-        println!("Error: data directory doesn't exist");
-        return;
-    }
+    ensure!(
+        data_dir_path.exists(),
+        "Error: data directory doesn't exist"
+    );
 
-    BreezServices::init_logging(&data_dir, None).expect("Failed to init logging");
+    BreezServices::init_logging(&data_dir, None)?;
 
     let persistence = CliPersistence { data_dir };
     let history_file = &persistence.history_file();
 
-    let rl = &mut Editor::<()>::new().unwrap();
+    let rl = &mut Editor::new()?;
+    rl.set_helper(Some(CliHelper {
+        hinter: HistoryHinter {},
+        colored_prompt: format!("\x1b[1;32m{}\x1b[0m", "sdk> "),
+    }));
```

Comment:
What does this do?

Reply:
Well, it colors the prompt. It's a sort of "being color green" / "end color" type of char sequence for terminals.

Yea its kind of weird. I can add a comment to clarify or just remove it, let me know.

Reply:
It's one thing the new version of `rustyline` supports, so I figured we could use it.

Reply:
OK, it seems the color is added again in the highlighter? https://github.com/breez/breez-sdk/pull/442/files#diff-7e62d977711c2b08170e3927acd4e04382b392b03dc4326a5e6bdaa3c646e6daR77

Reply:
That's separate, it highlights the autocompleted text in a different shade of white.

Based on the CLI history, e.g. if you called `node_info` before, as you start typing `node` it will autofill `_info` with the different highlight. Pressing right arrow auto-completes the command.